src/transformers/generation/utils.py

```diff
@@ -2264,6 +2264,7 @@ def sample(
 
         >>> # set pad_token_id to eos_token_id because GPT2 does not have a EOS token
         >>> model.config.pad_token_id = model.config.eos_token_id
+        >>> model.generation_config.pad_token_id = model.config.eos_token_id
```

Comment:
This is done in https://github.com/huggingface/transformers/pull/20804/files, although it is not merged.
We can remove the change here, but thanks.